templates/html_tag/html_tag.component.inc

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @file
+ * Contains component file.
+ */
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function europa_preprocess_html_tag(&$variables, $hook) {
+  $headers = ['h1', 'h2', 'h3', 'h4', 'h5'];
```

Comment:
@haringsrob We are using GrumPHP in Atomium but we should also use it here indeed.